x/emissions/keeper/inference_synthesis/network_regrets.go

```diff
@@ -330,7 +359,18 @@ func GetCalcSetNetworkRegrets(args GetCalcSetNetworkRegretsArgs) error {
 
 	// Recalculate topic initial regret
 	if len(workersRegrets) > 0 {
-		updatedTopicInitialRegret, err := CalcTopicInitialRegret(workersRegrets, args.EpsilonTopic, args.PNorm, args.CNorm)
+		params, err := args.K.GetParams(args.Ctx)
+		if err != nil {
+			return errorsmod.Wrapf(err, "GetCalcSetNetworkRegrets error getting params")
+		}
```

Comment:
You can also avoid this query by adding the `InitialRegretQuantile` and `PnormSafeDiv` in the args since we already have the params in `CloseReputerNonce`.

Also, please follow the pattern:
`PnormSafeDiv` -> `PNormSafeDiv`

Reply:
added the first point in my commit!

Reply:
fixed second in 2nd commit